Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/AlternativeIdService.java

```diff
@@ -0,0 +1,50 @@
+package edu.asu.conceptpower.app.core.impl;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.core.ConceptTypesService;
+import edu.asu.conceptpower.app.core.ConceptTypesService.ConceptTypes;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+@Service
+public class AlternativeIdService {
+
+    @Autowired
+    private ConceptTypesService conceptTypesService;
```

Comment:
I just realize that, but this class needs an interface and be moved into the correct package.